elements/castbar.lua

```diff
@@ -183,6 +188,11 @@ local UNIT_SPELLCAST_INTERRUPTED = function(self, event, unit, spellname, _, cas
 		return
 	end
 
+	local text = castbar.Text
+	if(text) then
+		text:SetText(INTERRUPTED)
+	end
+
 	castbar.casting = nil
```

Comment:
Fair enough, setting the bar to max can be done in PostUpdate should the layout _really_ want that for the looks (the other way around is harder).